api/controllers/ResumeController.ts

```diff
@@ -1,11 +1,14 @@
-import { JsonController, UseBefore, Post, UploadedFile, BadRequestError } from 'routing-controllers';
+import {
+  JsonController, Get, UseBefore, ForbiddenError, Post, UploadedFile, BadRequestError,
+} from 'routing-controllers';
 import * as path from 'path';
+import PermissionsService from '../../services/PermissionsService';
 import StorageService from '../../services/StorageService';
 import { UserAuthentication } from '../middleware/UserAuthentication';
 import ResumeService from '../../services/ResumeService';
 import { UserModel } from '../../models/UserModel';
 import { AuthenticatedUser } from '../decorators/AuthenticatedUser';
-import { File, MediaType, UpdateResumeResponse } from '../../types';
+import { File, MediaType, GetResumesListResponse, UpdateResumeResponse } from '../../types';
 
```

Comment:
nit: let's call this `GetVisibleResumesResponse` to match the API name

Reply:
edited